CStdString.h

```diff
@@ -4,7 +4,7 @@
 //
 //      !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
 //      This is a "facelifted" version of Joe's original code, made compatible
-//      with C++17 compilers (by Szabolcs Sz�sz, in Aug. 2019).
+//      with C++17 compilers (by Szabolcs Szász, in Aug. 2019).
```

Comment:
I think what happened here was that this file was originally encoded with a Windows codepage (Windows-1252), and when I edited it in either Visual Studio or VS Code, it was changed to UTF-8.  Originally these characters were replaced with invalid UTF-8 sequences, but I restored them back to the correct UTF-8 sequences.